aea/protocols/scaffold/message.py

```diff
@@ -38,21 +38,21 @@ def __str__(self):
             """Get string representation."""
             return self.value
 
-    def __init__(self, oef_type: Optional[Type] = None,
-                 **kwargs):
+    def __init__(self, oef_type: Optional[Type] = None, **kwargs):
         """
         Initialize.
 
         :param oef_type: the type of message.
         """
         super().__init__(type=oef_type, **kwargs)
-        assert self.check_consistency(), "MyScaffoldMessage initialization inconsistent."
+        self.consistent = self.check_consistency()
+#       assert self.check_consistency(), "MyScaffoldMessage initialization inconsistent."
 
     def check_consistency(self) -> bool:
         """Check that the data is consistent."""
         try:
             raise NotImplementedError
-        except (AssertionError, ValueError):
+        except (AssertionError, ValueError, NotImplementedError):
```

Comment:
I think we don't want the consistency check to pass with False on NotImplementedError. I think we want it to throw the error.